specs/gloas/beacon-chain.md

```diff
@@ -1047,6 +1144,146 @@ def process_operations(state: BeaconState, body: BeaconBlockBody) -> None:
     for_ops(body.payload_attestations, process_payload_attestation)
 ```
 
+##### Deposit requests
+
+###### New `get_index_for_new_builder`
+
+```python
+def get_index_for_new_builder(state: BeaconState) -> BuilderIndex:
+    for index, builder in enumerate(state.builders):
+        if builder.withdrawable_epoch <= get_current_epoch(state) and builder.balance == 0:
```

Comment:
Could you remind me of the reason that we want to reuse indices of exited builders? I'm curious as we don't reuse exited validator's index.